src/test/features/claim/routes/check-and-send.ts

```diff
@@ -87,13 +88,15 @@ describe('Claim issue: check and send page', () => {
           .expect(res => expect(res).to.be.successful.withText('<a href="/claim/reason" class="bold">Change <span class="visuallyhidden">why you believe you’re owed the money:</span></a>'))
```

Comment:
`.withText(...)` accepts a vararg of strings, so you can just comma-separate them

Reply:
updated please have a look

Reply:
LGTM